cosmwasm/contracts/crosschain-registry/src/state.rs

```diff
@@ -0,0 +1,25 @@
+use cosmwasm_std::Addr;
+use cw_storage_plus::{Item, Map};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+// CONTRACT_ALIAS_MAP is a map from a contract alias to a contract address
+pub const CONTRACT_ALIAS_MAP: Map<&str, String> = Map::new("contract_alias_map");
+
+// CHAIN_TO_CHAIN_CHANNEL_MAP is a map from a source<>destination chain pair to its respective channel id
+pub const CHAIN_TO_CHAIN_CHANNEL_MAP: Map<(&str, &str), String> =
+    Map::new("chain_to_chain_channel_map");
+
+// CHANNEL_TO_CHAIN_CHAIN_MAP is a map from a channel id on a source chain to its respective destination chain
+pub const CHANNEL_TO_CHAIN_CHAIN_MAP: Map<(&str, &str), String> =
```

Comment:
can we rename to `CHANNEL_ON_CHAON_CHAIN_MAP` ?

"to" makes it seem like we are mapping a channel to a chain

Reply:
https://github.com/osmosis-labs/osmosis/pull/4320/commits/740d4bdd8cb18a6dea53f9469988c97aa0a30d7b